packages/typespec-ts/test/integration/azurecore.spec.ts

```diff
@@ -101,6 +101,36 @@ describe("Azure Core Rest Client", () => {
     }
   });
 
+  it("should list with parameters", async () => {
+    try {
+      const result = await client.path("/azure/core/basic/parameters").get({
+        queryParameters: { another: "Second" },
+        body: { inputName: "Madge" }
+      });
+      assert.strictEqual(result.status, "200");
+    } catch (err) {
+      assert.fail(err as string);
+    }
+  });
+
+  it("should get first item as page item", async () => {
+    try {
+      const result = await client.path("/azure/core/basic/first-item").get();
+      assert.strictEqual(result.status, "200");
```

Comment:
you need to assert the detail for result.

Reply:
I have updated the test file as you said.